frontend/src/app/sessions/session/tiling-window-manager/tiling-window-manager.component.html

```diff
@@ -0,0 +1,75 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<div class="flex gap-0.5">
+  <ng-container *ngFor="let session of sessions">
+    <div
+      class="flex flex-col"
+      [ngClass]="{
+        height: (fullscreenService.isFullscreen$ | async) === false,
+        'h-screen': fullscreenService.isFullscreen$ | async
+      }"
+      [style.width]="session.width + 'px'"
+    >
+      <div
+        class="flex items-center justify-between gap-2 rounded-t bg-slate-100 p-1"
+      >
+        <span>
+          {{ session.version?.tool?.name }} {{ session.version?.name }},
+          {{ session.type }}
+          <span *ngIf="session.type === 'readonly'"
+            >(project {{ session.project!.name }})</span
+          >
+          {{ session.id }}
+        </span>
+
+        <div class="flex items-center">
+          <mat-icon
+            *ngIf="session.index !== 0"
+            class="cursor-pointer text-lg text-gray-600 hover:text-gray-800"
+            (click)="onLeftClick(session)"
+          >
+            arrow_back
+          </mat-icon>
+
+          <mat-icon
+            *ngIf="session.index !== sessions.length - 1"
+            class="cursor-pointer text-lg text-gray-600 hover:text-gray-800"
+            (click)="onRightClick(session)"
+          >
+            arrow_forward
+          </mat-icon>
+        </div>
+      </div>
+
+      <div class="relative grow">
+        <div
+          class="iframe-overlay"
+          *ngIf="isValidResizeState(this.resizeState)"
+        ></div>
+        <iframe
```

Comment:
We should disable the "selection" of the whole iframe (if possible). This happens sometimes accidentally. 

https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/b11ae22c-465b-4f6a-acfc-c9c3416fd3ac

Reply:
I added a `class="select-none"` to the `div` containing the floating and tiling window manager, so now selection should no longer be possible.